### PR TITLE
clean up tests so they pass locally + in the CI

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by jsancio** [comment]
LGTM! :shipit:

File: README.rst [hunk 1]
```diff
@@ -101,17 +101,37 @@ Running Tests
...
+directory to one of those two and follow the instructions.
+
+
+Initialization
+#######
+
+The `config` integration tests use static config files. To run these tests
+make sure you set owner only permissions on these files:
+
+:code:`chmod 600 cli/tests/data/dcos.toml`
```
Comment (kensipe) [inline README.rst]: what prevents us from checking in the file with the correct permissions.  I like that we documented this... however I would love for it to be automated.

If there is reason to not check in a file with the correct attributes, then can we have a script that automate this?

File: README.rst [hunk 2]
```diff
@@ -101,17 +101,37 @@ Running Tests
...
+:code:`echo "dcos/cluster/url dcos.snakeoil.mesosphere.com" >> /etc/hosts`
+
 
 Running
 #######
 
-Tox will run unit and integration tests in both Python environments using a
-temporarily created virtualenv.
+Tox will run unit and integration tests in Python 3.4 using a temporarily
+created virtualenv.
```
Comment (kensipe) [inline README.rst]: is it ok to check in a `.python-version` with the project which can 1) provide documentation of expectation and auto use the correct version for those using `pyenv`

**Comment by kensipe** [comment]
I have a few questions but otherwise lgtm.

File: README.rst (hunk 1 above)
Comment (tamarrow) [inline README.rst]: great idea, I added to the `make test` command

Comment (msabramo) [inline README.rst]: Also see #969, which does it in Python, in case someone doesn't notice the `Makefile` and runs `tox` directly (like I did 😄 )

**Pull request merged**